docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-kubernetes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade YugabyteDB Anywhere using Kubernetes
```

Comment:
I'm finding this a bit confusing, I might be wrong here. 

Considering that YugabyteDB Anywhere will be install on Kubernetes using Helm. Should this be something like: Upgrade YugabyteDB Anywhere installed on Kubernetes  
or   
Upgrade Kubernetes installation of YugabyteDB Anywhere.

Or all three mean the same? If yes, then we can just stick to what we have.